packages/accounts-password/password_tests.js

```diff
@@ -55,6 +55,9 @@ if (Meteor.isClient) (() => {
     Meteor.call('removeSkipCaseInsensitiveChecksForTest', value);
 
   const createUserStep = function (test, expect) {
+    Meteor.logout(expect(_ => {
+      test.equal(Meteor.user(), null);
+    }));
```

Comment:
I think we should await the logout here so the assertion runs as part of the same test execution.

Something like this:

``` js
await new Promise((resolve, reject) => {
  Meteor.logout(err => {
    if (err) return reject(err);
    resolve();
  });
});

test.equal(Meteor.user(), null);
```